modules/client-configuration/pages/registration-bootstrap.adoc

```diff
@@ -69,6 +69,14 @@ For more about custom certificates, see xref:administration:ssl-certs.adoc[].
 ====
 
 
+[WARNING]
+====
```

Comment:
We need to change this as well, in the same way we need changes for *6 OS. Maybe we should have the same message here, but with the list of affected OS.